datamol/io.py

```diff
@@ -83,6 +83,8 @@ def read_sdf(
     mol_column: str = None,
     include_private: bool = False,
     include_computed: bool = False,
+    sanitize: bool = True,
+    strictParsing: bool = True,
```

Comment:
Sorry, I should have been more explicit: can you use `strict_parsing` instead of `strictParsing` for consistency?